packages/itwinui-css/scripts/createComponent.js

```diff
@@ -3,7 +3,7 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 const fs = require('fs');
-const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('shared/copyrightLinter');
+const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('./copyrightLinter');
```

Comment:
```suggestion
const { copyrightBannerScss, copyrightBannerHtml, copyrightBannerJs } = require('../../../scripts/copyrightLinter');
```

Reply:
no, the path is correct. they both live in the same folder

Reply:
ah you're right. we really need to combine these two files 😅